### ISSUE TITLE
Search won't work with partial words in RTL

### ISSUE THREAD

**Issue by PostMidnight** [NodeBB/NodeBB#7430]
Hi Developers,
I am using postgresql.
To reproduce, type in partial word and search.
Results: both quick and advance search won't show any results.
In English however, partial words may be search.
Let me know should you need additional information.
JJ.

**Comment by julianlam** [comment]
Please follow issue submission guidelines and report database type and version. Are you using any search plugins, etc? This contextual information is important to share.

**Comment by PostMidnight** [comment]
Hi @julianlam 
I am using postgesql (v.10). My code is synced with master. with the default search plugin (DB SEARCH v.3.0.6). 
PG v9 is not working as well, so it looks like the issue is elsewhere.
Thanks for looking into this!
JJ.

**Comment by julianlam** [comment]
So pgSQL v10 works better, and doesn't suffer from these issues?

**Comment by pitaj** [comment]
Provide the exact search terms you're using, what you get, and what you expected to get.

**Comment by PostMidnight** [comment]
Hi PG v9 and v10 works in the same way.
I am not using English. in RTL languages (Arabic, hebrow, *) A partial word (for example, "pita") won't generate results, while "pitaj" will.
Looking at the plugin, it seems that index is performed only for certain supported languages. Mine is not supported, could that be the issue?

**Comment by PostMidnight** [comment]
Hi @julianlam and @pitaj, I reviewed the code a bit, as follows:

First, I added the language code into: var languageLookup = { (line 22 of db-search), followed by 
setting the index language in ACP. Pushing the "change language" button crashed nodebb.

The reason: postgres.js: line 75: the function call had language spelled incorrectly.

Unfortunately, the revised code failed in line 81 of postgres.js
			db.pool.query('DROP INDEX "idx__searchtopic__content"', next);

with err = Error: index "idx__searchtopic__content" does not exist at Connection.parseE.
I can't go further as I don't know exactly why the index is not there - maybe @BenLubar knows.

Any help will be appreciated thanks!
